### PR TITLE
Add --btf option

### PR DESCRIPTION
**Pull request by qjerome**
This PR proposes a fix for issue #176

### CODE REVIEW

**Review by alessandrod — CHANGES_REQUESTED** [review]
Looks good, thanks!

I think we also need to move this behind the flag https://github.com/aya-rs/bpf-linker/blob/f451f76ddfbfd57ef2c3c7536e0d6b7a894f9be7/src/linker.rs#L452

**Comment by qjerome** [comment]
Just realized I broke some tests, now needing this flag to be set... Fixing everything up.

**Comment by qjerome** [comment]
@alessandrod : your change request has been addressed in the last commit

File: tests/btf/assembly/exported-symbols.rs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // assembly-output: bpf-linker
-// compile-flags: --crate-type bin -C link-arg=--emit=obj -C debuginfo=2 -C link-arg=--log-level=debug -C link-arg=--log-file=/tmp/linker-lol.log
+// compile-flags: --crate-type bin -C link-arg=--emit=obj -C debuginfo=2 -C link-arg=--log-level=debug -C link-arg=--log-file=/tmp/linker-lol.log -C link-arg=--btf
```
Comment (alessandrod) [inline tests/btf/assembly/exported-symbols.rs:2]: Oops can you please remove --log-level and --log-file here? I didn't mean to commit those 😅

Comment (qjerome) [reply tests/btf/assembly/exported-symbols.rs]: done

**Review by alessandrod — APPROVED** [review]
lgtm thank you!

**Pull request merged**